tests/Collapse/CollapseTest.php

```diff
@@ -112,10 +112,12 @@ public function testSetInnerHitsOverridesExistingValue(): void
         $collapse->setInnerHits($innerHits1);
         $collapse->addInnerHits($innerHits2);
 
-        $this->assertCount(2, $collapse->getParam('inner_hits'));
-        $this->assertIsArray($collapse->getParam('inner_hits'));
-        $this->assertEquals($innerHits1, $collapse->getParam('inner_hits')[0]);
-        $this->assertEquals($innerHits2, $collapse->getParam('inner_hits')[1]);
+        $innerHits = $collapse->getParam('inner_hits');
+
+        $this->assertCount(2, $innerHits);
+        $this->assertIsArray($innerHits);
+        $this->assertEquals($innerHits1, $innerHits[0]);
+        $this->assertEquals($innerHits2, $innerHits[1]);
```

Comment:
This has been changed to not confuse PHPStan, because below there is this call:

```php
$this->assertInstanceOf(InnerHits::class, $collapse->getParam('inner_hits'));
```
So without this change, this was the error:

```
 ------ -------------------------------------------------------------------------------------------------------------------------------------------
  Line   tests/Collapse/CollapseTest.php
 ------ -------------------------------------------------------------------------------------------------------------------------------------------
  125    Call to method PHPUnit\Framework\Assert::assertInstanceOf() with 'Elastica\\Collapse\\InnerHits' and array will always evaluate to false.
 ------ -------------------------------------------------------------------------------------------------------------------------------------------
```